docs/operator-manual/monitor-mode.md

```diff
@@ -57,39 +50,25 @@ spec:
     - NET_ADMIN
 ```
 
-## Transitioning policy mode
-
-It's worth noting that certain attributes can be updated on policies
-once they have been deployed.
-
-For security purposes, a user with UPDATE permissions on policy
-resources can make the policy more restrictive, so that:
+## Changing policy mode
 
-- Transitioning the `mode` of an existing `ClusterAdmissionPolicy` or
-  `AdmissionPolicy` from `monitor` to `protect` is allowed.
+It's worth noting that you can update certain attributes on policies
+that are already deployed.
```

Comment:
Maybe this phrase could be removed altogether.

Reply:
Thanks, done.